Dockerfile

```diff
@@ -79,37 +76,49 @@ RUN conan export conan/mqtt_cpp \
 FROM os AS release
 
 LABEL author="mtconnect" description="Docker image for the latest Production MTConnect C++ Agent"
+ARG BIN_DIR=/usr/bin
+ARG MTCONNECT_DIR=/etc/mtconnect
+
+ENV MTCONNECT_DIR=$MTCONNECT_DIR
 
 # install ruby for simulator
-RUN apt-get update && apt-get install -y ruby
+RUN apt-get update && apt-get install -y ruby zip
 
 # change to a new non-root user for better security.
 # this also adds the user to a group with the same name.
 # --create-home creates a home folder, ie /home/<username>
 RUN useradd --create-home agent
```

Comment:
We might also want to merget these two `RUN` commands into one, although it would be a bit less readable.